### PR TITLE
Added Perigee Keyboard

### PR DESCRIPTION
**Pull request by FreshCourageTake**
## Description
Added Perigee keyboard with default and VIA key mappings

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/keebmaker/perigee/rev1/rules.mk [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/rev1/rules.mk:18]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = no            # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/keebmaker/perigee/readme.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make perigee/rev1:default
+
+Install examples:
+
+    make perigee/rev1:default:dfu         # For Perigee rev1
+    
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/readme.md:20]:
```suggestion
# Perigee

![Perigee]

A 50% (12x5) ortholinear keyboard kit made and sold by KeebMaker LLC. 

* Keyboard Maintainer: [Kyle Lee](https://github.com/klee813)
* Hardware Supported: Perigee PCB rev1
* Hardware Availability: [keebmaker.com]

Make example for this keyboard (after setting up your build environment):

    make perigee/rev1:default

Flashing example for this keyboard:

    make perigee/rev1:default:flash
    
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Missing image.

File: keyboards/keebmaker/perigee/perigee.h [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+** Include headers specific to keyboard revision **
+**************************************************/
+#include "rev1.h"
+
+
+/**************************************************
+** Layout macros aliases common to all revisions **
+**                                               **
+**      Name of Alias     Matrix Name:           **
+******* ----------------- ------------------- ****/
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/perigee.h:32]:
```suggestion
#include "rev1.h"

```

File: keyboards/keebmaker/perigee/keymaps/via/rules.mk [hunk 4]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes 
+RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/keymaps/via/rules.mk:2]:
```suggestion
VIA_ENABLE = yes
RGBLIGHT_ENABLE = yes
```

File: keyboards/keebmaker/perigee/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+ * |------+------+------+------+------+------+                      +------+------+------+------+------+------|
+ * | Ctrl | Ctrl | Alt  | GUI  | One  | Spc  |                      | Spc  |  Two | Left | Down |  Up  |Right |
+ * `------------------------------------------                      ------------------------------------------'
+ */
+[_QWERTY] = LAYOUT_ortho_5x12(
+  KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,      KC_5,            KC_6,    KC_7,      KC_8,    KC_9,    KC_0,    KC_BSPC,
+  KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,      KC_T,            KC_Y,    KC_U,      KC_I,    KC_O,    KC_P,    KC_DEL,
+  KC_ESC,  KC_A,    KC_S,    KC_D,    KC_F,      KC_G,            KC_H,    KC_J,      KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+  KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,      KC_B,            KC_N,    KC_M,      KC_COMM, KC_DOT,  KC_SLSH, KC_ENT,
+  KC_LCTL, KC_LCTL, KC_LALT, KC_LGUI, MO(ONE), KC_SPC,          KC_SPC,  MO(TWO), KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/keymaps/via/keymap.c:53]:
```suggestion
  KC_LCTL, KC_LCTL, KC_LALT, KC_LGUI, MO(1), KC_SPC,          KC_SPC,  MO(2), KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT
```

File: keyboards/keebmaker/perigee/keymaps/default/config.h [hunk 6]
```diff
@@ -0,0 +1,17 @@
+/* Copyright 2022 KeebMaker
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/keymaps/default/config.h:1]: This file can be deleted.

File: keyboards/keebmaker/perigee/keymaps/default/rules.mk [hunk 7]
```diff
@@ -0,0 +1 @@
+RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/keymaps/default/rules.mk:1]:
```suggestion
RGBLIGHT_ENABLE = yes
```

File: keyboards/keebmaker/perigee/keymaps/default/readme.md [hunk 8]
```diff
@@ -0,0 +1 @@
+# The default Perigee layout - largely based on the Preonic's
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/keymaps/default/readme.md:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]
`.noci` should also be deleted.

**Comment by FreshCourageTake** [comment]
> `.noci` should also be deleted.

.noci has been deleted and all other files cleaned up.

File: keyboards/keebmaker/perigee/perigee.h [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#include "rev1.h"
+
+#define LAYOUT_perigee_grid LAYOUT_ortho_5x12
```
Comment (noroadsleft) [inline keyboards/keebmaker/perigee/perigee.h:24]:
```suggestion
```

File: keyboards/keebmaker/perigee/readme.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+
+A 50% (12x5) ortholinear keyboard kit made and sold by KeebMaker LLC. 
+
+* Keyboard Maintainer: [Kyle Lee](https://github.com/klee813)
+* Hardware Supported: Perigee PCB rev1
+* Hardware Availability: [keebmaker.com]
+
+Make example for this keyboard (after setting up your build environment):
+
+    make perigee/rev1:default
```
Comment (noroadsleft) [inline keyboards/keebmaker/perigee/readme.md:13]:
```suggestion
    make keebmaker/perigee/rev1:default
```

File: keyboards/keebmaker/perigee/readme.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+
+Flashing example for this keyboard:
+
+    make perigee/rev1:default:flash
```
Comment (noroadsleft) [inline keyboards/keebmaker/perigee/readme.md:17]:
```suggestion
    make keebmaker/perigee/rev1:default:flash
```

File: keyboards/keebmaker/perigee/readme.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
+# Perigee
+
+![Perigee]
```
Comment (noroadsleft) [inline keyboards/keebmaker/perigee/readme.md:3]: There should be an image/photo here representative of the board.

```
![Perigee](https://imgur.com/someURL.jpg)
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/keebmaker/perigee/readme.md (hunk 12 above)
Comment (FreshCourageTake) [reply keyboards/keebmaker/perigee/readme.md]: All concerns have been addressed

File: keyboards/keebmaker/perigee/info.json [hunk 13]
```diff
@@ -0,0 +1,74 @@
+{
+    "keyboard_name": "Perigee",
+    "maintainer": "kylelee",
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/info.json:3]:
```suggestion
    "keyboard_name": "Perigee",
    "manufacturer": "KeebMaker",
    "maintainer": "kylelee",
    "usb": {
        "vid": "0x4338",
        "pid": "0x13F6",
        "device_version": "0.0.1"
    },
```

File: keyboards/keebmaker/perigee/config.h [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4338
+#define PRODUCT_ID      0x13F6
+#define MANUFACTURER    KeebMaker
+#define PRODUCT         Perigee
+
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/config.h:27]:
```suggestion
```

File: keyboards/keebmaker/perigee/readme.md [hunk 15]
```diff
@@ -0,0 +1,19 @@
+# Perigee
+
+![Perigee](https://imgur.com/a/wZjrkkx)
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/readme.md:3]:
```suggestion
![Perigee](https://i.imgur.com/Uzxhmujh.jpeg)
```

File: keyboards/keebmaker/perigee/rev1/config.h [hunk 16]
```diff
@@ -0,0 +1,22 @@
+/*
```
Comment (fauxpark) [inline keyboards/keebmaker/perigee/rev1/config.h:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**